lib/policyEvaluator/utils/actionMaps.ts

```diff
@@ -228,15 +228,15 @@ const actionMapMetadata = {
 };
 
 const actionMapScuba = {
```

Comment:
rename this as well for consistency and to notice easily where it is used ?

Reply:
Let's rename everything. We were not planning to use these ARNs in Zenko for now as we use roles but indeed at some point it will be needed and exposed, so better use `SUR` everywhere...